src/common/obj_bencher.cc

```diff
@@ -799,6 +799,11 @@ int ObjBencher::seq_read_bench(int seconds_to_run, int num_objects, int concurre
   }
 
   runtime = ceph_clock_now() - data.start_time;
+  if(runtime <= 1) {
```

Comment:
oh sorry. there will be no need to update the iops again if runtime==1 because it has get correct result already. and commit has been updated.